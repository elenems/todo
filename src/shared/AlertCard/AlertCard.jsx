import React from "react";
import Typography from "@material-ui/core/Typography";
import Box from "@material-ui/core/Box";
import { MdWarning } from "react-icons/md";
export default function AlertCard(props) {

  return (
    <Box
      style={{
        padding: "20px 16px",
        border: "1px solid",
        marginBottom: "12px"
      }}
      color="#f50057"
      display="flex"
      alignItems="center"
      justifyContent="space-between"
      className='reveal card'
    >
      <Typography variant="h6" component="p">
        {props.error}
      </Typography>
      <div style={{ fontSize: "48px", lineHeight: "0em" }}>
        <MdWarning color="#f50057" />
      </div>
    </Box>
  );
}
