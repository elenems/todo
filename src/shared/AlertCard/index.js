import AlertCard from './AlertCard.jsx';
export default AlertCard;