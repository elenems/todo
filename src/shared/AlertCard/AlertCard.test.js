import React from 'react';
import {shallow} from 'enzyme';
import AlertCard from '../AlertCard';
describe('Alert card', ()=>{
 it('Renders', ()=>{
     const wrapper = shallow(<AlertCard />).debug();
     expect(wrapper).toMatchSnapshot();
 })
})