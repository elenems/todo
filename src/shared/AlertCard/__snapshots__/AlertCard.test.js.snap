// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert card Renders 1`] = `
"<Styled(MuiBox) style={{...}} color=\\"#f50057\\" display=\\"flex\\" alignItems=\\"center\\" justifyContent=\\"space-between\\" className=\\"reveal card\\">
  <WithStyles(ForwardRef(Typography)) variant=\\"h6\\" component=\\"p\\" />
  <div style={{...}}>
    <MdWarning color=\\"#f50057\\" />
  </div>
</Styled(MuiBox)>"
`;
